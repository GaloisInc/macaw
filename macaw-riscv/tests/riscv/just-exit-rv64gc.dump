
just-exit-rv64gc.exe:     file format elf64-littleriscv


Disassembly of section .text:

000000000001010c <main>:
   1010c:	1141                	addi	sp,sp,-16
   1010e:	e422                	sd	s0,8(sp)
   10110:	0800                	addi	s0,sp,16
   10112:	0001                	nop
   10114:	6422                	ld	s0,8(sp)
   10116:	0141                	addi	sp,sp,16
   10118:	8082                	ret
