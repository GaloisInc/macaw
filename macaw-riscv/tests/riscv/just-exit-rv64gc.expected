R { fileEntryPoint = Nothing
  , funcs = [(0x1010c, [(0x1010c,14)])]
  , ignoreBlocks = []
  }
